Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml

```diff
@@ -0,0 +1,63 @@
+Parser:
+  Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
+  Version: '0.4.1'
+  LastUpdated: Jun 18, 2024
+Product:
+  Name: Microsoft Windows Events Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.4'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Microsoft Windows Sysmon process terminate events (event 5) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema. 
+ParserName: ASimProcessEventTerminateMicrosoftSysmonWindowsEvent
+EquivalentBuiltInParser: _ASim_ProcessEvent_TerminateMicrosoftSysmonWindowsEvent
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: | 
+    let parser = (disabled:bool = false) {
+        let parser_WindowsEvent=(disabled:bool=false){
+            WindowsEvent
+            | where not(disabled)
+            | where Provider == "Microsoft-Windows-Sysmon" and EventID == 5
+            | extend
+                EventProduct = "Security Events",
+                ActorUsername = tostring(EventData.User),
+                TargetProcessName = tostring(EventData.Image),
+                TargetProcessId = tostring(EventData.ProcessId),
+                TargetProcessGuid = tostring(EventData.ProcessGuid),
+                TargetUsername = tostring(EventData.User)
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', ''),
```

Comment:
![image](https://github.com/user-attachments/assets/ed973608-0fb6-4f4b-91c0-2c9c2d10fee8)

Reply:
I think we can skip this based on documentation

Reply:
you mean targetusernametype?